content/en/load-testing/Manual/060-test-development.md

```diff
@@ -12,11 +12,11 @@ description: >
 
 ### The Environment: your Favorite IDE
 
-XLT test cases are usually developed in Java, as JUnit test cases. This means you can work locally, in your IDE of choice. You will probably start out with one of our [sample test suites](../../test-suites), which come as Maven projects ready to import to your IDE of choice, build and run, and adjust those test cases to your needs, probably also adding new test cases. If you need to do some functional testing, test data may be added.
+XLT test cases are usually developed in Java, as JUnit test cases. This means you can work locally, in your IDE of choice. You will probably start out with one of our [sample test suites](../../test-suites), which come pre-configured as Maven projects ready to imported, built and run in your IDE of choice. You'll then want to adjust some test cases to your needs, and probably also add new ones. If you need to do some functional testing, test data may be added.
```

Comment:
"ready to **be** imported..." or "ready to import, build..."? This seems not quite right...

Reply:
Fixed.